linux/set_launcher_icon

```diff
@@ -7,6 +7,7 @@
 APPDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 if [ -w "$APPDIR"/zotero.desktop ]; then
 	sed -i -e "s@^Icon=.*@Icon=$APPDIR/chrome/icons/default/default256.png@" "$APPDIR"/zotero.desktop
+	cp ${APPDIR}/zotero.desktop ${XDG_DATA_HOME:-${HOME}/.local/share}/applications/
```

Comment:
XDG_DATA_HOME is from https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html